simulator/src/main/resources/BlockAsDirException/1/1.blk

```diff
@@ -0,0 +1 @@
+asdasd
```

Comment:
Is this ^^ for a test or something?

Reply:
Many of the `blk` files under `blockAsDirExample` are listed as empty. Should those be part of the PR?

Reply:
> Is this ^^ for a test or something?

Yes is to produce a ParseException on the BlockAsDir BlockStreamManager

Reply:
I have updated the Files for the `BlockAsDir` test, it should not be empty now 👍 

✅  Done!